src/python/yapykaldi/asr.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import (print_function, division, absolute_import, unicode_literals)
+from builtins import *
```

Comment:
If you have `python-future` installed then it will work. It is the standard way of futurizing a python script to be compatible with 2 and 3. We could use the specific imports, but as this PR is still a prototype and not ready be merged to master, I let this be.

Reply:
Can you add that to the requirements then plz?

Reply:
Done in a new PR.